test_marginal is flakey

```
pymc_experimental/tests/test_marginal_model.py::test_marginalized_deterministic_and_potential - AssertionError: 
Arrays are not almost equal to 7 decimals

Mismatched elements: 2 / 5 (40%)
Max absolute difference: 2.3841858e-07
Max relative difference: 9.66977e-08
 x: array([ 1.6125156,  2.3487637,  2.4792917, -1.0471407,  2.4656074],
      dtype=float32)
 y: array([ 1.6125154,  2.3487637,  2.4792914, -1.0471407,  2.4656076],
      dtype=float32)
FAILED pymc_experimental/tests/distributions/test_continuous.py::TestGenExtremeClass::test_logcdf - AssertionError: 
Arrays are not almost equal to 2 decimals
{'mu': array(0.01, dtype=float32), 'sigma': array(1., dtype=float32), 'xi': array(0.99, dtype=float32), 'value': array(-1., dtype=float32)}
Mismatched elements: 1 / 1 (100%)
Max absolute difference: 1.86426127
Max relative difference: 0.00016986
 x: array(-10973.14, dtype=float32)
 y: array(-10975.01)
```

--- a/pymc_experimental/tests/test_marginal_model.py
+++ b/pymc_experimental/tests/test_marginal_model.py
@@ -296,6 +296,8 @@ def test_not_supported_marginalized():
 
 @pytest.mark.filterwarnings("error")
 def test_marginalized_deterministic_and_potential():
+    rng = np.random.default_rng(299)
+
     with MarginalModel() as m:
         x = pm.Bernoulli("x", p=0.7)
         y = pm.Normal("y", x)
@@ -306,7 +308,7 @@ def test_marginalized_deterministic_and_potential():
     with pytest.warns(UserWarning, match="There are multiple dependent variables"):
         m.marginalize([x])
 
-    y_draw, z_draw, det_draw, pot_draw = pm.draw([y, z, det, pot], draws=5)
+    y_draw, z_draw, det_draw, pot_draw = pm.draw([y, z, det, pot], draws=5, random_seed=rng)
     np.testing.assert_almost_equal(y_draw + z_draw, det_draw)
     np.testing.assert_almost_equal(det_draw, pot_draw - 1)
 

